src/modules/m_nick.c

```diff
@@ -294,14 +294,13 @@ void nick_identify (Nick *nptr, User *uptr __unused, char *all)
         NoticeToUser(nptr,"Syntax: \2NICK IDENTIFY \037password\037\2");
         return;
     }   
-        
     user = find_user(nptr->nick);
     if (!user) {
         NoticeToUser(nptr,"You are not registered");
         return;
     }   
-        
-    if (user->authed == 1) {
+    
+    if (nptr->svid != NULL) {
```

Comment:
Note that this will prevent users from re-authenticating to a different account except by reconnecting. Is that the intent?

Reply:
It's not, I do remove the svid when one changes their nick: https://github.com/xiu/child/blob/svid/src/parseline.c#L871

It creates another issue though where one looses their svid even though they weren't auth on their previous nick but the nick was registered. I'll put more thought into that. For the purpose of this PR, we could still rely on `user->authed` since accounts and nicks are not yet separated.